test/benchmark/BenchmarkIngest_test.go

```diff
@@ -109,24 +109,28 @@ func BenchmarkIngest(b *testing.B) {
 	testLimit := samplesCount * int(timestampsCount)
 
 	b.StartTimer()
-	for i := 0; i < b.N; i++ {
-		rowsAdded, err := runTest(i, appender, timestamps, sampleTemplates, refs,
-			testConfig.AppendOneByOne, testConfig.BatchSize, testConfig.Verbose, testConfig.ValidateRawData)
 
-		if err != nil {
-			b.Fatal(err)
-		}
+	var ec int
+	withTimer(b, fmt.Sprintf("BenchmarkIngest-%d -> Table: %s", b.N, tsdbPath), func() {
```

Comment:
Why do you need both `b.StartTimer() / b.StopTimer()` and `withTimer`?

Reply:
It measures a test cycle time without using performance metrics (simple calc)
Example output:
```
BenchmarkIngest_test.go:315: BenchmarkIngest-200 -> Table: tsdb-BenchmarkIngest-200-2018-10-04T15_41_18_03_00
			Elapsed time: 3438 millisecond
```